README.md

```diff
@@ -465,6 +465,217 @@ node followerDiff.js
+
+Note: If running on RH8 or Rocky Linux 8, you may adapt the given
+docker commands with `crictl` instead.
+
+
+### Step 1: Generate the digests database
+
+Start by scanning all raft sessions using `verifyBucketSproxydKeys`,
+to generate a digests database in order to have a more efficient scan
+of followers later on, that does not require an entire listing of the
+leader again.
+
+Provide the environment variables **LISTING_DIGESTS_OUTPUT_DIR** and
+enable **NO_MISSING_KEY_CHECK** to speed up the listing.
+
+Example:
+
+```
+DIGESTS_PATH=~/followerDiff-digests-$(date -I);
+docker run \
+--net=host \
+--rm \
+-e 'BUCKETD_HOSTPORT=127.0.0.1:9000' \
+-e 'RAFT_SESSIONS=1,2,3,4,5,6,7,8' \
+-e 'LISTING_DIGESTS_OUTPUT_DIR=/digests' \
+-v "${DIGESTS_PATH}:/digests" \
+-e 'NO_MISSING_KEY_CHECK=1' \
+registry.scality.com/s3utils/s3utils:1.13.0 \
+node verifyBucketSproxydKeys.js \
+| tee -a verifyBucketSproxydKeys.log
+```
+
+### Step 2: Copy the digests database to each active metadata stateful server
+
+Example:
+
+```
+USER=root
+for HOST in storage-1 storage-2 storage-3 storage-4 storage-5; do
+    scp -r ${DIGESTS_PATH} ${USER}@${HOST}:followerDiff-digests
+done
+```
+
+### Step 3: Save the list of raft sessions to scan on each stateful server
+
+The following script computes the list of raft sessions to scan for
+each stateful server, and outputs the result as a series of ssh
+commands to execute to save the list on each active stateful. It
+requires the 'jq' command to be installed where it is executed.
+
+It starts with all listed raft sessions for each active stateful, but
+removes the raft sessions on each active stateful on which it is the
+leader.
+
+You may save the following script to a file before executing it with "sh".
+
+```
+#!/bin/sh
+
+### Config
+
+BUCKETD_ENDPOINT=localhost:9000
+USER=root
+RS_LIST="1 2 3 4 5 6 7 8"
+
+
+### Script
+
+JQ_SCRIPT_EACH='
+.leader.host as $leader
+| .connected[]
+| select(.host != $leader)
+| { host, $rs }'
+
+JQ_SCRIPT_ALL='
+group_by(.host)
+| .[]
+| .[0].host as $host
+| "echo \(map(.rs) | join(" ")) > /tmp/rs-to-scan"
+| @sh "ssh -n \($user)@\($host) \(.)"'
+
+for RS in $RS_LIST; do
+    curl -s http://${BUCKETD_ENDPOINT}/_/raft_sessions/${RS}/info \
+    | jq --arg rs ${RS} "${JQ_SCRIPT_EACH}";
+done | jq -rs --arg user ${USER} "${JQ_SCRIPT_ALL}"
+```
+
+Executing this script gives on the standard output, the list of ssh
+commands to run to save the list of raft sessions to scan in a
+temporary file on each active stateful member, for example:
+
+```
+ssh -n 'root'@'storage-1' 'echo 3 4 5 7 > /tmp/rs-to-scan'
+ssh -n 'root'@'storage-2' 'echo 1 2 3 5 6 7 8 > /tmp/rs-to-scan'
+ssh -n 'root'@'storage-3' 'echo 1 2 3 4 5 6 8 > /tmp/rs-to-scan'
+ssh -n 'root'@'storage-4' 'echo 1 2 3 4 5 6 7 8 > /tmp/rs-to-scan'
+ssh -n 'root'@'storage-5' 'echo 1 2 4 6 7 8 > /tmp/rs-to-scan'
+```
+
+Check that what you get looks like the above, then execute those
+commands on the shell (e.g. by piping with `| bash -x`).
```

Comment:
maybe adding the full command to be run is better here

Reply:
Agree, I can add the command to run